egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -507,23 +594,35 @@ def combine_models(dir, num_iters, models_to_combine, egs_dir,
     context_opts = "--left-context={lc} --right-context={rc}".format(
         lc=left_context, rc=right_context)
 
+    if use_multitask_egs:
+        combine_egs = "scp:{0}/combine.egs.1.scp".format(egs_dir)
+    else:
+        combine_egs = "ark:{0}/combine.egs".format(egs_dir)
+
+    multitask_egs_opts = common_train_lib.get_multitask_egs_opts(
+                             egs_dir,
+                             egs_prefix="combine.",
+                             use_multitask_egs=use_multitask_egs)
     common_lib.run_job(
         """{command} {combine_queue_opt} {dir}/log/combine.log \
                 nnet3-combine --num-iters=40 \
                 --enforce-sum-to-one=true --enforce-positive-weights=true \
                 --verbose=3 {raw_models} \
-                "ark,bg:nnet3-copy-egs {context_opts} \
-                    ark:{egs_dir}/combine.egs ark:- | \
+                "ark,bg:nnet3-copy-egs {context_opts} {multitask_egs_opts} \
+                {egs_rspecifier} ark:- |{extra_egs_copy_cmd} \
```

Comment:
It is input to functions and it sets to "" in function definition. 
It could be helpful to pass extra eg-copy command to modify egs. 
Do you want me to remove this option or we can keep it for future?